be/src/storage/lake/vacuum.cpp

```diff
@@ -72,6 +73,33 @@ std::future<Status> completed_future(Status value) {
     return p.get_future();
 }
 
+bool should_retry(const Status& st, int64_t attempted_retries) {
+    if (attempted_retries >= config::lake_vacuum_retry_max_attempts) {
+        return false;
+    }
+    Slice message = st.message();
```

Comment:
mapped slow-down request limit to ResourceExhaustedError, may check this type and determine retry or not.